saleor/graphql/order/types.py

```diff
@@ -175,6 +176,8 @@ class Order(CountableDjangoObjectType):
         TaxedMoney,
         description='The sum of line prices not including shipping.')
     status_display = graphene.String(description='User-friendly order status.')
+    can_finalize = graphene.Boolean(
+        description='Informs if an order draft can be complete.')
```

Comment:
I'd rephrase a little bit: "Informs whether a draft order can be finalized (turned into a regular order)."